onlineapi/src/main/kotlin/com/codingblocks/onlineapi/models/APIModel.kt

```diff
@@ -102,6 +102,32 @@ data class Course(
 
     }
 
+    @Type("hbcontents")
+    data class CoeChallenge(
+        val name: String,
+        @Relationship("problem")
+        val content: included?=null
+
+    ):BaseModel()
+
+    @Type("details")
+    data class detailsClass(
+        val constraints: String?="None",
+        val explanation: String?="None",
+        val input_format: String?="None",
+        val sample_input: String?="None",
```

Comment:
Remove snake case and use camelCase 
` @JsonProperty("input_formatr")` to serialize the field